lib/dialects/mysql/query.js

```diff
@@ -92,7 +92,22 @@ class Query extends AbstractQuery {
       this.handleInsertQuery(data);
 
       if (!this.instance) {
-        result = data[this.getInsertIdField()];
+        // handle bulkCreate AI primiary key 
+        if (
+          data.constructor.name === 'ResultSetHeader'
```

Comment:
https://github.com/sequelize/sequelize/pull/6475#discussion_r75575249
It is easier than importing the class as the library is loaded lazily and I think we would have to do a require into one of the libraries internal files to get a reference.